src/webui/api/appcontroller.cpp

```diff
@@ -68,6 +69,17 @@ void AppController::versionAction()
     setResult(QBT_VERSION);
 }
 
+void AppController::buildInfoAction()
+{
+    const QJsonObject versions = {
```

Comment:
Do you mind adding the 64/32 bit fields to it?
for example: 
```c++
#if defined(__x86_64__) || defined(_M_X64)
    {"bitness", 64},
#else
    {"bitness", 32},
#endif
```

Then you can show it the title: `qBittorrent v4.2.0alpha Web UI (64-bit)`.

Reply:
WebUI is web application running in client browser. Why do you want to append server-side "bitness" to its title. It's confusing.

Reply:
>Why do you want to append server-side "bitness" to its title. It's confusing.

Following this logic we should omit the Libraries tab altogether... 
Of course a WebUI user wants to know server-side libraries info and its bitness.

Reply:
Or maybe just show it in Libraries tab, not in title.

Reply:
>Following this logic we should omit the Libraries tab altogether...

No. Just let them know that this is server info.

Reply:
I would use Qt macro Q_PROCESSOR_X86_64 to check "bitness".

Reply:
>Then maybe QT_POINTER_SIZE?

Sounds good! It even can be coded as: `{"bitness", (QT_POINTER_SIZE * 8)},`
maybe later we should update all related code to use `QT_POINTER_SIZE`.